var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -44,7 +51,7 @@ class PyPyprecice(PythonPackage):
     depends_on("py-mpi4py", type=("build", "run"), when="+mpi")
```

Comment:
I won't think the `wheel` dep is needed

Reply:
Also, the `py-mpi4py` package is always required, so I would remove the variant.